README.md

```diff
@@ -62,6 +62,8 @@ npm run dev [component-name(s)]
 
 ### Testing
 
+#### 🕸 Legacy tests ([Web Component Tester](https://github.com/Polymer/web-component-tester))
```

Comment:
😆 Love the emojis

Reply:
Do you want to bump this below the new tests?

Reply:
Ah yes let's do that.